dirty_cat/target_encoder.py

```diff
@@ -44,8 +45,14 @@ class TargetEncoder(BaseEstimator, TransformerMixin):
         present during transform (default is to raise). When this parameter
         is set to 'ignore' and an unknown category is encountered during
         transform, the resulting one-hot encoded columns for this feature
-        will be all zeros. In the inverse transform, an unknown category
-        will be denoted as None.
+        will be all zeros.
```

Comment:
I think that the sentence that you removed is actually not about missing values, but about a category that has not been seen at train. I should probably be put back, maybe rephrasing so that it is more clear, something like:
```suggestion
        will be all zeros. In the inverse transform, an unknown category
        (not seen during training) will be denoted as None.
```

Reply:
I removed it because TargetEncoder does not have an inverse_transform() function. 

```
>>> te.inverse_transform(xx)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'TargetEncoder' object has no attribute 'inverse_transform'
```